social.info.yml

```diff
@@ -1,7 +1,7 @@
 name: Social
 type: profile
 description: 'Open Social install profile.'
-core_version_requirement: ^8.8
+core_version_requirement: ^8.8 || ^9
```

Comment:
We did `8.9 - 9.1` in other `.info.yml` files, shouldn't this be the same?

Reply:
That's what you've seen in composer.json and it shouldn't be in any .info.yml files.

As the Drupal documentation dictates us to use `core_version_requirement: ^8.8 || ^9` 
See - https://www.drupal.org/node/3070687

Reply:
My bad, you're right.